tests/Mono.Android-Tests/Xamarin.Android.Net/AndroidMessageHandlerTests.cs

```diff
@@ -39,20 +39,29 @@ protected override HttpMessageHandler CreateHandler ()
 #if NET
 		[Test]
 		[TestCaseSource (nameof (DecompressionSource))]
-		[Retry (5)]
+		// Disabled because it doesn't exist in NUnitLite, uncomment when/if we switch to full NUnit
+		// When we can use it, replace all the Console.WriteLine calls with Assert.Warn
+		// [Retry (5)]
 		public async Task Decompression (string urlPath, string encoding, string jsonFieldName)
 		{
 			// Catch all the exceptions and warn about them or otherwise [Retry] above won't work
 			try {
-				DoDecompression (urlPath, encoding, jsonFieldName);
+				int count = 0;
+				// Remove the loop when [Retry] can be used
+				while (count < 5) {
+					if (await DoDecompression (urlPath, encoding, jsonFieldName)) {
+						return;
+					}
+					count++;
+				}
 			} catch (Exception ex) {
-				Assert.Warn ("Unexpected exception thrown");
-				Assert.Warn (ex.ToString ());
+				Console.WriteLine ("Unexpected exception thrown");
+				Console.WriteLine (ex.ToString ());
```

Comment:
Should this instead use `TestContext.Out.WriteLine()`?

@dellis1972, @jonathanpeppers?

Reply:
Yes, I think that is the only way it would make it into a log.

Reply:
neither `TestContext.Out.WriteLine` nor `TestContext.WriteLine` work. It appears that NUnitLite doesn't have that feature either :(

Reply:
Ok here: https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=7395571&view=artifacts&pathAsName=false&type=publishedArtifacts

run-Mono.Android.NET_Tests-Debug.binlog has lines like:
```
02-28 17:33:35.485  7425  7445 I NUnit   : JniValueMarshalerContractTests`1.CreateObjectReferenceArgumentState_DefaultValue 
02-28 17:33:35.486  7425  7445 I NUnit   :     Passed
02-28 17:33:35.486  7425  7445 I NUnit   : JniValueMarshalerContractTests`1.CreateReturnValueFromManagedExpression 
02-28 17:33:35.494  7425  7445 I DOTNET  : # jonp: expected: JniValueMarshaler_String_ContractTests
```
Ok, for this test Console.WriteLine() will work fine:
https://github.com/xamarin/java.interop/blob/77800dda83c2db4d90b501c00069abc9880caaeb/tests/Java.Interop-Tests/Java.Interop/JniValueMarshalerContractTests.cs#L267

It wouldn't work in our MSBuild tests, but Device Tests w/ NUnitLite this is the only option.